File: check_spring_server/src/main/java/com/weather/bigdata/it/app/check_spring_server/service/SendHelper.scala
package com.weather.bigdata.it.app.check_spring_server.service

import java.util

trait SendHelper {
  def beat(heartbeatTitle: String): Boolean
  def text(access_tokens: util.Set[String], title: String, text: String, outurl: String, pushtype: Integer): Boolean
  def text(token: String, title: String, text: String, outurl: String, pushtype: Integer): Boolean
  def text(token: String, title: String, text: String, pushtype: Integer): Boolean
}

File: src/main/java/com/weather/bigdata/it/app/check_spring/service/SendHelper.java
package com.weather.bigdata.it.app.check_spring.service;

import java.util.Set;

public interface SendHelper {
    Boolean beat(String heartbeatTitle);

    Boolean text(Set<String> access_tokens,String title, String text, String outurl, Integer pushtype);
    Boolean text(String token,String title,String text, String outurl, Integer pushtype);
    Boolean text(String token,String title,String text, Integer pushtype);
}
